tests/unit_test/data_processor/data_processor_test.py

```
@@ -7,13 +7,16 @@
 import urllib
```

Ruff (0.8.2)

`urllib` imported but unused

Remove unused import: `urllib`

(F401)